python/ray/autoscaler/aws/config.py

```diff
@@ -138,6 +138,11 @@ def _configure_subnet(config):
         [s for s in ec2.subnets.all() if s.state == "available"],
         reverse=True,  # sort from Z-A
         key=lambda subnet: subnet.availability_zone)
+    if not subnets:
+        raise Exception(
+            "No subnets found, try manually creating an instance in "
+            "your specified region to populate the list of subnets "
+            "and trying this again.")
```

Comment:
I'm not sure why the subnets list is empty until you do this; I ran into this problem when trying to use the script in a region I hadn't used before (us-west-2).